package.json

```diff
@@ -29,5 +30,7 @@
     "not IE 11"
   ],
   "dependencies": {
+    "@rails/actioncable": "^7.2.100",
```

Comment:
Won't this depend on the rails version the host app is using?

Reply:
You are suggesting I update the version to be a range?

Reply:
We want to allow rails 7.1 and rails 8.0 (eventually), right? I think some kind of range would be preferable.